typings/third-party-defs.d.ts

```diff
@@ -1,12 +1,8 @@
 declare var BlobBuilder: any;
-declare var errorWrapper: any;
 declare var MIDI: any;
 declare var Guppy: any;
 declare var Sk: any;
 declare var safari: any;
 declare var system: any;
```

Comment:
This and the next three are internal, right? (and can be removed)

Reply:
These variables are present here: 

1. [safari](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/domain/utilities/BrowserCheckerService.js#L30)
2. [system](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/expressions/ExpressionEvaluatorService.js#L106)

There is no declaration for them so we need to declare them in custom types but they can be moved to custom-oppia-defs instead of third-party-defs. Is that ok?

Reply:
safari should just become window.safari, I think.

system looks like a possible error? Do you know where it is defined? I think that needs a bit of digging and resolution -- it is definitely not an external type.

Reply:
No, I tried to search. There is no definition I found except in the comment mentioned in the same file [here](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/expressions/ExpressionEvaluatorService.js#L37).

I will update safari to window.safari and check if it works fine.